docs/testrunner-toolkit/configuration.md

```diff
@@ -66,7 +66,7 @@ sauce:
 When you launch a tunnel, you can accept the tunnel identifier name that Sauce Labs generates for your account (e.g., `{SL-username}_tunnel_id`) or specify a name in the launch command:
 
 ```
-bin/sc -u {SL-username} -k {SL-access_key} -i {tunnel_identifier}
+./sc -u {SL-username} -k {SL-access_key} -i {tunnel_name}
```

Comment:
Hey @kimsauce --- thanks for making this change. I think we can also edit the following sentence, since the config property has been renamed from `id` to `name` and no longer conflicts with the UI. Maybe just:

_`saucectl` expects this `tunnel_name` value in the `tunnel.name` property of your config file._

Reply:
Thank you! Done.